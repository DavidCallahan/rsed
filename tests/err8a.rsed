source="test25.input"
input $source
foreach for 4
	print $CURRENT
end
close $source
required "model" error "start of model section"
foreach all
	print $CURRENT
end
