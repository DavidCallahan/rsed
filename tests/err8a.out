processor	: 0
vendor_id	: GenuineIntel
cpu family	: 6
model		: 45
script 7: failed required pattern: "model": start of model section
